eng/Directory.Build.Common.props

```diff
@@ -12,6 +12,7 @@
 
     <!-- Client in this context is any library that is following the new Azure SDK guidelines -->
     <IsClientLibrary Condition="'$(IsClientLibrary)' == '' and $(MSBuildProjectName.StartsWith('Azure.'))">true</IsClientLibrary>
+    <IsFunctionsLibrary Condition="'$(IsFunctionsLibrary)' == '' and $(MSBuildProjectName.StartsWith('Microsoft.Azure.WebJobs.Extensions.'))">true</IsFunctionsLibrary>
```

Comment:
For my info Functions == WebJobs? Or is this just a general Extensions support?

Reply:
Functions runs on WebJobs and that specific package prefix is what the functions extensions packages are required to use.